src/SprykerEco/Yves/Braintree/Theme/default/components/molecules/credit-card/DEPRECATED.md

```diff
@@ -0,0 +1 @@
+This component is deprecated. Use molecule braintree-credit-card instead.
```

Comment:
This component is deprecated. Use molecule('braintree-credit-card', 'Braintree') instead.
`----`
fix all `deprecated` messages

Reply:
fixed